Mesh_3/doc/Mesh_3/Mesh_3.txt

```diff
@@ -706,6 +706,24 @@ The resulting mesh is shown in \cgalFigureRef{figureliver_3d_image_mesh}.
 Cut view of a 3D mesh produced from a segmented liver image. Code from subsection \ref Mesh_3_subsection_examples_3d_image generates this file.
 \cgalFigureEnd
 
+\subsubsection Mesh_3DomainsFrom3DImagesWithWeights Domains From Segmented 3D Images, with Weights
+
+When a segmented image is given as input, the generated mesh surface sometimes sticks too closely
+to the voxels surface, causing an aliasing effect.
+A solution to generate a smooth and accurate output surface was described by Stalling et al in
+\cgalCite{stalling1998weighted}. It consists in generating a second input image, made
+of integer coefficients called "weights". This image can be generated using
```

Comment:
```suggestion
\cgalCite{stalling1998weighted}. It consists in generating a second input image, made
of integer coefficients called "weights", and use those weights to define smoothing domain boundaries. The image of weights can be generated using
```

Reply:
That is a suggestion. I am not completely sure of the wording "image of weights".